app/assets/stylesheets/application.scss

```diff
@@ -45,36 +44,22 @@
   }
 }
 
-table {
```

Comment:
:(

This means a lot more classes in the HTML but I understand @robinwhittleton's concerns (https://github.com/alphagov/govuk-lint/pull/59#issuecomment-243806623).

We should really look into a bootstrap thing that allows us to include mixins, would make our lives a lot easier.